src/app/controllers/view_controller.spec.ts

```diff
@@ -64,4 +64,64 @@ describe('ViewController', function () {
     const drawSecondDom = drawerSpy.calls.mostRecent().args[0];
     expect(drawSecondDom.contentBlock.content).toBe(`real content ${drawSecondDom.contentBlock.contentPath}`);
   });
+
+  it('initializes the contents of specified content-blocks', function () {
```

Comment:
it seems to me like there is a lot of duplicated code here, so i suggest to refactor it